### PR TITLE
GF-60, GF-53 - Persist the redux store, Deadline Step error

### PR DESCRIPTION
**Pull request by iamdavidpetric**

### CODE REVIEW

**Review by MIKALAKE — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -1,16 +1,23 @@
 import React from 'react';
 import { Provider } from 'react-redux';
 import ReactDOM from 'react-dom/client';
+import { PersistGate } from 'redux-persist/integration/react';
 
 import './index.css';
```
Comment (iamalextodoran) [inline src/index.js]: This is a relative path. Where is this supposed to go?

File: src/process/redux/store.js [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
+  storage,
+  whitelist: ['user', 'projects'],
+  timeout: null
+};
+
+const persistedReducer = persistReducer(persistConfig, reducer);
+
+const logger = createLogger({
+  collapsed: true
+});
```
Comment (iamalextodoran) [inline src/process/redux/store.js]: Leave it a one liner

**Review by iamalextodoran — CHANGES_REQUESTED** [review]
You also solved your bug (on Jira), put the ticket id in here.

**Comment by iamdavidpetric** [comment]
> You also solved your bug (on Jira), put the ticket id in here.

File: src/index.js (hunk 1 above)
Comment (iamdavidpetric) [reply src/index.js]: bottom

**Review by iamalextodoran — APPROVED** [review]

**Pull request merged**